README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
Well it looks like ``.bower.json`` is auto generated once package is installed. I have tested with ``.bower.json`` descriptor and it starts breaking files resolving for me. The main reason is even bower.json does not exist .bower.json is being generated without any usefull information. For example [ded/script.js](https://github.com/ded/script.js) package is being installed through bower but has only package.json , if I do include .bower.json as a file descriptor it's being resolved incorrectly because ``.bower.json`` file always exists.

I would suggest to stick with ``bower.json`` for now, atleast until there is a way to skip description file, if "main" attribute is not being found in it.